openmetadata-ui/src/main/resources/ui/src/pages/SearchSettingsPage/SearchSettingsPage.tsx

```diff
@@ -0,0 +1,327 @@
+              />
+            </Col>
+            <Col className="global-setting-card">
+              <Typography.Text className="global-setting-card__content">
+                {t('label.use-natural-language-search')}
+              </Typography.Text>
+              <Switch
+                checked={searchConfig?.globalSettings?.useNaturalLanguageSearch}
+                className="m-l-xlg global-setting-card__action"
+                data-testid="use-natural-language-search-switch"
+                disabled={isUpdating}
+                onChange={() =>
+                  handleUpdateSearchConfig({
+                    enabled:
+                      !searchConfig?.globalSettings?.useNaturalLanguageSearch,
+                    field: 'useNaturalLanguageSearch',
+                  })
+                }
+              />
+            </Col>
+            {globalSettings.map(({ key, label }) => (
+              <Col className="global-setting-card" key={key}>
+                <GlobalSettingItem
+                  label={label}
+                  value={searchConfig?.globalSettings?.[key] ?? 0}
+                  onUpdate={(value) =>
+                    handleUpdateSearchConfig({
+                      field: key,
+                      value,
+                    })
+                  }
+                />
+              </Col>
+            ))}
+          </Row>
+          <Row className="term-boosts-section m-t-md" gutter={[0, 16]}>
+            <Col span={24}>
+              <div className="d-flex items-center justify-between m-b-md">
+                <Typography.Text className="text-md font-medium">
+                  {t('label.configure-term-boost')}
+                </Typography.Text>
+                <Button
+                  data-testid="add-term-boost"
+                  type="primary"
+                  onClick={handleAddNewTermBoost}>
+                  {t('label.add-term-boost')}
+                </Button>
+              </div>
+              <div
+                className="d-flex items-center gap-2 flex-wrap term-boosts-container"
+                data-testid="term-boosts">
+                {searchConfig?.globalSettings?.termBoosts?.map((termBoost) => (
+                  <TermBoostComponent
+                    key={termBoost.field}
+                    termBoost={termBoost}
+                    onDeleteBoost={handleDeleteTermBoost}
+                    onTermBoostChange={handleTermBoostChange}
+                  />
+                ))}
+              </div>
+            </Col>
+          </Row>
+        </Col>
+      </Row>
+
+      {/* <FilterConfiguration /> */}
+
+      <Row className="p-x-lg p-b-md" gutter={[16, 16]}>
+        {settingCategoryData?.map((data) => (
+          <Col key={data.key} span={6}>
+            <SettingItemCard
+              isButton
+              className="search-setting-card"
+              data={data}
+              key={data.key}
+              onClick={() => handleViewDetailClick(data.key)}
```

Comment:
Can we have key from callback function and rewrite this as 
```suggestion
              onClick={handleViewDetailClick}
```

Reply:
Yes, made the changes accordingly.